src/coreComponents/python/modules/pygeosx_tools_package/pygeosx_tools/file_io.py

```diff
@@ -0,0 +1,53 @@
+
+import os
+import numpy as np
+
+
+def save_tables(grid, properties, table_root='./tables', axes=['x', 'y', 'z', 't']):
+    """
+    Saves a set of tables in GEOSX format
+    Notes: The shape of these arrays should match the length of each axis in the specified order
+           The output directory will be created if it does not exist yet
+
+    Args:
+        grid (list): A list of numpy ndarrays defining the table axes
+        properties (dict): A dict of numpy ndarrays defning the table values
+        table_root (str): The root path for the output directory
+        axes (list): A list of names for each potential axis (default = ['x', 'y', 'z', 't'])
+    """
+    os.makedirs(table_root, exist_ok=True)
+    for ii, d in zip(range(len(grid)), axes):
+        np.savetxt('%s/%s.csv' % (table_root, d),
```

Comment:
Just curious if we would eventually want to roll this functionality into the time history stuff, or make time history IO callable as a free function from Python without needing an associated GEOSX instance. Certainly we want to make it easy to read in the time history files in Python. I just would prefer to avoid a proliferation of file types.

Reply:
Don't worry about it, I haven't had the time to work on this myself!

This is the code I typically use to read/write the structured table functions in python, and I could see it being called in either place.  The general use case I have in mind here is as follows:

1) Run the simulation
2) Grab the desired GEOSX variables using pygeosx
3) Interpolate the values onto a regular grid
4) Write these values into a table function, so that they can be used elsewhere